config/rbac/auth_proxy_service.yaml

```diff
@@ -1,14 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: elasticsearch-operator-metrics
   labels:
-    control-plane: controller-manager
-  name: controller-manager-metrics-service
-  namespace: system
+    name: elasticsearch-operator
+  name: elasticsearch-operator-metrics
+  namespace: openshift-operators-redhat
```

Comment:
Same here. We can customize the namespace from the kustomization.yaml

Reply:
Good catch!